readme.md

```diff
@@ -1,10 +1,4 @@
-# 🔥 Aprendiendo Git con Hugo 🔥
+# 🔥 Arruinando el codigo 🔥
```

Comment:
Nope, we don't make that here

![image](https://media.giphy.com/media/njYrp176NQsHS/giphy.gif)